tests/watchdog_test.go

```diff
@@ -42,14 +43,30 @@ var _ = Describe("[sig-compute]Watchdog", decorators.SigCompute, func() {
 
 	Context("A VirtualMachineInstance with a watchdog device", func() {
 
-		It("[test_id:4641]should be shut down when the watchdog expires", decorators.Conformance, func() {
+		It("[test_id:4641]should be shut down when the watchdog expires", decorators.Conformance, decorators.WgS390x, func() {
+
 			vmi := libvmops.RunVMIAndExpectLaunch(
 				libvmifact.NewAlpine(libvmi.WithWatchdog(v1.WatchdogActionPoweroff)), 360)
 
 			By("Expecting the VirtualMachineInstance console")
 			Expect(console.LoginToAlpine(vmi)).To(Succeed())
 
-			By("Killing the watchdog device")
+			// Watchdog module need to be loaded as it will not be loaded by default for s390x
+			if libnode.GetArch() == "s390x" {
```

Comment:
In test - We don't need the if condition when ensuring the watchdog module is loaded. The command can be executed for all architectures, including x86, without any issues. If the module is already loaded, the command will simply return without any effect